drivers/dai/intel/ssp/ssp.c

```diff
@@ -1980,12 +1985,12 @@ static int dai_ssp_parse_tlv(struct dai_intel_ssp *dp, const uint8_t *aux_ptr, s
 		case SSP_LINK_CLK_SOURCE:
 #ifdef CONFIG_SOC_SERIES_INTEL_ADSP_ACE
```

Comment:
Should this be changed as well? #ifdef SSP_IP_VER >= SSP_IP_VER_1_5

Reply:
yes, good catch, thanks :)